src/content/h1/tools/aether/readme.md

```diff
@@ -1,15 +1,219 @@
+  <figcaption>
+    <p>Aether loaded with a file</p>
+  </figcaption>
+</figure>
+
+The BSP list selects which BSP you want to perfrom your actions on. With a BSP selected you have the option to Edit that BSPs' lightmap bitmap or export the data you need in 3ds MAX.
+
+If the Edit button is unavailable then your BSP has no lightmap bitmap associated with it, meaning you probably haven't debug lightmapped it yet.
+
+Hit the export button now.
+
+The options here should be pretty self explanatory, but you'll need at least the BSP and Lightmaps options selected for this tutorial. If you don't have all of your bitmaps exported already hit either All for all the bitmaps in all of the BSPs' shaders, or Diffuse only for just the diffuse textures. If you have objects in your map that you want to cast shadows or emit light then you need the Marker Export options checked too. Overwrite should be checked unless you've edited bitmaps for example, but you'll definately need it if you've recompiled your BSP as the lightmap UV's will be different. Hit export and Aether will output all of the info you need into your scenarios relative data directory.
+
+**3ds MAX - importing BSP**
+Next you need to import your BSP into 3ds MAX. Import the textured mesh .obj file using the settings provided on this page. Aether does export a material (.mtl) file with the BSP, which should also be imported by putting the filename into the material box. So now you should have your BSP imported, but it probably looks completely grey at the moment. You could go through each material and set them to show in viewport but the MAXscript supplied with Aether does this for you. Select the "MAXscript" tab and hit the "MAXScript button". If you don't have the script set to load on startup, hit Run Script and select it. To actually show the script interface you will need to reselect it in the Utilities drop down list, even if it is already shown.
+
+With your BSP mesh selected, press the Fix Materials button. This will set all of its materials to show in viewport and will set all of the diffuse bitmaps' Alpha Source value to None (Opaque), which stops the material from being rendered too white. You should be left with a correctly rendered BSP. 
+
+**3ds MAX - importing objects**
+
+To import your objects, you are going to need TheGhosts [GBXModel](http://hce.halomaps.org/index.cfm?fid=1972) model importer, so install that first. In the Aether script utility there is a Marker Tools button. Press it and a window will pop up which is what we are going to use to place all of our objects. Hit the Open Marker File button and select one of the aemk files you exported from Aether earlier. Once you have done so, the Markers list will be populated with the object types used in the map. Look at the first object in the scenery list. It's handy to have your scenario open in guerilla at this point so that you can easily find out what gbxmodel you need to open, as well as what bitmaps you need to extract.
```

Comment:
I know it was originally written this way on modacity, but might as well fix some grammar while we're at it: TheGhosts => TheGhost's.

Also the link to the gbxmodel importer should probably be written as `[gbxmodel importer](...)` to help build association in people's mind to the gbxmodel tag, which is consistently lowercase in c20, and to show that the link destination is related to a tool and not just the tag.

"aemk" looks like a typo rather than a file type. I suggest enclosing it with a code tag like `.aemk`.

Guerilla should be upper case and can be a link to the Guerilla page, e.g. `[Guerilla][]`.